tests/org.x.xeyes.yml

```diff
@@ -65,3 +65,23 @@ modules:
           url: https://httpbingo.org/base64/MS4wLjAK
           version-pattern: (\d[\d\.]+\d)
           url-template: https://httpbingo.org/status/404
+
+  - name: boost
+    sources:
+      - type: archive
+        url: https://dl.bintray.com/boostorg/release/1.75.0/source/boost_1_75_0.tar.bz2
+        sha256: "0000000000000000000000000000000000000000000000000000000000000000"
+        x-checker-data:
+          type: anitya
+          project-id: 6845
+          url-template: https://dl.bintray.com/boostorg/release/$version/source/boost_$major_$minor_$patch.tar.bz2
```

Comment:
Template includes `_` into placeholder name and tries to substitute `$major_`. You need to put it in braces, like `${major}_`